### PR TITLE
15230 Certificate of Restoration

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#15230

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-filer/src/entity_filer/filing_processors/restoration.py [hunk 1]
```diff
@@ -34,8 +34,14 @@ def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: F
...
-    name_request_json = restoration_filing.get('nameRequest')
-    business_info.set_legal_name(business.identifier, business, name_request_json)
+
+    if name_request_json := restoration_filing.get('nameRequest'):
+        business_info.set_legal_name(business.identifier, business, name_request_json)
+        if nr_number := name_request_json.get('nrNumber', None):
+            filing_meta.restoration = {
+                **filing_meta.restoration,
+                'nrNumber': nr_number
+            }
```
Comment (vysakh-menon-aot) [inline queue_services/entity-filer/src/entity_filer/filing_processors/restoration.py:44]: name request is not a required object

File: legal-api/src/legal_api/services/business_details_version.py [hunk 2]
```diff
@@ -233,17 +233,21 @@ def get_business_revision_obj(transaction_id, business):
         return business_revision
 
     @staticmethod
-    def get_business_revision_before_filing(filing_id, business_id) -> dict:
-        """Consolidates the business info of the previous filing."""
```
Comment (vysakh-menon-aot) [inline legal-api/src/legal_api/services/business_details_version.py:237]: `get_business_revision_before_filing` was not in use. removing this function

**Review by argush3 — APPROVED** [review]

File: legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py [hunk 3]
```diff
@@ -891,11 +891,11 @@ def test_unpaid_filing(session, client, jwt):
                     }
       },
      HTTPStatus.OK, '2017-10-01'
-    ),
+     ),
```
Comment (ChrisGabel23) [inline legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py:894]: This is really nitpicky, but I am curious if this extra space (repeated two times, below as well (918 and 942)) is needed or is a typo.

Comment (vysakh-menon-aot) [reply legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py:894]: It's auto formatted by vs code. If you see rest of the parameter object in this function you will see the same spacing

**Review by ChrisGabel23 — APPROVED** [review]

**Pull request merged**